src/app/config/config.default.ts

```diff
@@ -48,6 +48,11 @@ export const defaultConfig: Config = {
       enabled: false,
       turnOnPSUWhenExitingSleep: false,
     },
+    tplinkSmartPlug: {
+      enabled: false,
+      smartPlugIP: '127.0.0.1',
+      turnOnPowerWhenExitingSleep: false,
```

Comment:
```suggestion
```

I think we shouldn't have this value twice in the config (`turnOnPSUWhenExitingSleep` already exists)